stl/inc/__msvc_int128.hpp

```diff
@@ -132,8 +132,9 @@ struct
     template <size_t __m, size_t __n>
     static constexpr void _Knuth_4_3_1_M(
         const uint32_t (&__u)[__m], const uint32_t (&__v)[__n], uint32_t (&__w)[__n + __m]) noexcept {
-        _STL_INTERNAL_STATIC_ASSERT(__m <= size_t{(numeric_limits<int>::max)()});
-        _STL_INTERNAL_STATIC_ASSERT(__n <= size_t{(numeric_limits<int>::max)()});
+        constexpr auto _Int_max = size_t{(numeric_limits<int>::max)()};

```

Comment:
Nitpick, we conventionally avoid this pattern. Directly naming the type is simpler:

```suggestion
        constexpr size_t _Int_max{(numeric_limits<int>::max)()};
```

Occurs below.

Reply:
Would you believe me if I told you that your suggestion triggers `C4365: 'initializing': conversion from 'int' to 'const size_t', signed/unsigned mismatch` and mine does not?

Reply:
Woof. I can live with this as-is, then.